### PR TITLE
[ZH] Update `Performance_points`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Performance_points/zh.md [hunk 1]
```diff
@@ -1,123 +1,132 @@
...
-然后分数互相"权衡"来确保只有用户的最佳成绩取得最多的pp，即[*权重系统*](#权重系统)，目的是为了防止短时间多次在简单的谱面中取得低pp成绩降低玩家最佳成绩实际得到的pp。
+玩家在谱面中获得的表现分，还要通过一个[*权重系统*](#权重系统)加权，求和后才能得到玩家的总表现分。这个权重系统是为了防止玩家多次刷简单谱面，来获得不符合玩家实际水平的高表现分。加权后，玩家的总表现分主要由玩家的多个较高的成绩所决定。
 
-*注: 会根据玩过并留有成绩的Ranked谱面数量奖励少量的额外pp。*
+*注意：系统还会通过计算玩家游玩的上架谱面数量，得到少量的奖励表现分，并计入玩家的总表现分。*
 
 ### 权重系统
 
-权重系统是一个在算出一次游玩的成绩的全部pp后用到的简单公式。这个公式用于根据上述游玩在玩家的最佳成绩中的排行减少得到的pp。上述的公式如下所示：
+权重系统是一个简单的公式，用于计算玩家的总表现分。这个公式能够降低玩家的最好表现成绩榜上，比较靠后的成绩对总表现分的影响力。上述公式如下：
```
Comment (kj415j45) [inline wiki/Performance_points/zh.md:41]:
```suggestion
权重系统是一个用于计算玩家的总表现分的简单公式。这个公式能够降低玩家成绩榜上比较靠后的成绩对总表现分的影响。公式如下：
```

File: wiki/Performance_points/zh.md [hunk 2]
```diff
@@ -1,123 +1,132 @@
...
+### 精准 (Aim)
 
-*瞄准* 是一个衡量持续击中谱面中连续的物件的难度的主要标准。
+*精准 (Aim)* 是衡量玩家连续击打物件时，光标的移动精准度的指标。
 
-类似于 [缩圈速度](/wiki/Beatmapping/Approach_rate) 和特定的 [Mods](/wiki/Game_modifier)（即[Flashlight](/wiki/Game_modifier/Flashlight)，[Hidden](/wiki/Game_modifier/Hidden)和[Hard Rock](/wiki/Game_modifier/Hard_Rock)）使快速准确地移动光标的难度显著地提升，因此影响了成绩得到pp的数量。
+谱面的[缩圈速度 (AR)](/wiki/Beatmapping/Approach_rate)，和部分[游戏模组](/wiki/Game_modifier)（即 [Flashlight](/wiki/Game_modifier/Flashlight)、[Hidden](/wiki/Game_modifier/Hidden)、[Hard Rock](/wiki/Game_modifier/Hard_Rock) 这些模组）会显著地影响快速准确地移动光标的难度，从而影响谱面的表现分。
 
-在[osu!](/wiki/Game_mode/osu!)中，包含远距离[jump](/wiki/Beatmap/Pattern/Jump)的谱面的“瞄准”值会很高，于是通常得到的pp会很多。以此类推，包含很多hyperdash的[osu!catch](/wiki/Game_mode/osu!catch)谱面也会有类似的效果。瞄准在类似[osu!taiko](/wiki/Game_mode/osu!taiko) 和 [osu!mania](/wiki/Game_mode/osu!mania)的游戏模式中不会被考虑。
+在 [osu!](/wiki/Game_mode/osu!) 游戏模式中，含有较多的较远[跳](/wiki/Beatmap/Pattern/Jump)排列的谱面常被称为跳图，通常能在其上获得较多的表现分。同样地，在 [osu!catch](/wiki/Game_mode/osu!catch) 游戏模式中，含有较多的红果跳排列的谱面也能获得较多的表现分。[osu!taiko](/wiki/Game_mode/osu!taiko) 和 [osu!mania](/wiki/Game_mode/osu!mania) 游戏模式没有精准 (Aim) 指标。
```
Comment (kj415j45) [inline wiki/Performance_points/zh.md:53]:
```suggestion
在 [osu!](/wiki/Game_mode/osu!) 游戏模式中，含有较多的长距离[跳](/wiki/Beatmap/Pattern/Jump)排列的谱面常被称为跳图，通常能在其上获得较多的表现分。同样地，在 [osu!catch](/wiki/Game_mode/osu!catch) 中，含有较多的红果跳排列的谱面也能获得较多的表现分。[osu!taiko](/wiki/Game_mode/osu!taiko) 和 [osu!mania](/wiki/Game_mode/osu!mania) 则没有精准 (Aim) 指标。
```

File: wiki/Performance_points/zh.md [hunk 3]
```diff
@@ -1,123 +1,132 @@
...
 
-### 速度
+### 速度 (Speed)
 
-*速度* 是一个代表谱面物件速度的主要标准。
+*速度 (Speed)* 是衡量谱面物件密度的指标。
 
-短时间内包含了大量物件的谱面*速度*值很高。此时，*速度*越高，谱面难度就越高，也会得到更多的pp。
+短时间内出现大量物件的谱面，它的速度指标会很高。速度越高，谱面难度越高，也就能提供更多的表现分。
```
Comment (kj415j45) [inline wiki/Performance_points/zh.md:59]:
```suggestion
短时间内出现大量物件的谱面，它的速度指标会较高。而该值越高，代表谱面难度越高，也就能提供更多的表现分。
```

File: wiki/Performance_points/zh.md [hunk 4]
```diff
@@ -1,123 +1,132 @@
...
 
-因此诸如[Double Time](/wiki/Game_modifier/Double_Time)和[Half Time](/wiki/Game_modifier/Half_Time) 等Mod会由于pp算法显著影响谱面的*速度*。
+因此，诸如 [Double Time](/wiki/Game_modifier/Double_Time) 或 [Half Time](/wiki/Game_modifier/Half_Time) 这些影响谱面物件密度的模组，会显著改变谱面的速度指标，从而影响谱面的表现分。
```
Comment (kj415j45) [inline wiki/Performance_points/zh.md:61]:
```suggestion
如 [Double Time](/wiki/Game_modifier/Double_Time) 或 [Half Time](/wiki/Game_modifier/Half_Time) 模组会影响谱面物件密度，从而显著改变谱面的速度指标，进而影响谱面的表现分。
```

File: wiki/Performance_points/zh.md [hunk 5]
```diff
@@ -1,123 +1,132 @@
...
+### 精确 (Accuracy)
 
-*又见: [准确率](/wiki/Gameplay/Accuracy)*
+*参见：[准确率](/wiki/Gameplay/Accuracy)*
 
-*准确率* 是一个权衡玩家准时击中[物件](/wiki/Hit_object)能力的百分值；它也是一项和pp算法相关的衡量玩家在谱面中某项表现的标准。
+*精确 (Accuracy)* 是衡量玩家在[打击物件](/wiki/Hit_object)出现时，准时击打能力的指标，通常描述为一个百分值。这在衡量玩家的表现分算法里，也是一项比较重要的指标。
 
-准确率高的得分会得到很多的pp。一个取得 80% 准确率并[全连](/wiki/Full_combo) 的成绩有时相当于 95% 准确率的成绩。由于算法十分依赖于准确率，如Hidden，Hard Rock和Flashlight等Mod对高准确率的成绩得到的pp会增加很多。
+非常瞎眼的，高准确率的成绩，算法会给予其较高的表现分。因此，一个[全连](/wiki/Full_combo)，但是准确率仅为 80% 的成绩，它的含金量甚至只有准确率 95% 的成绩的 2/3。因为算法十分看重玩家的准确率，所以当玩家开启 Hidden、Hard Rock、Flashlight 这些模组，并打出高准确率的成绩时，能获得可观的表现分。
```
Comment (kj415j45) [inline wiki/Performance_points/zh.md:69]:
```suggestion
高准确率的成绩，算法会给予其较高的表现分。而一个[全连](/wiki/Full_combo)，但是准确率仅为 80% 的成绩，它的精确表现分甚至只有准确率 95% 的成绩的 2/3。因为算法十分看重玩家的准确率，所以当玩家开启 Hidden、Hard Rock、Flashlight 这些模组，并打出高准确率的成绩时，能获得可观的表现分。
```
`impressive` 译为 `非常瞎眼的`，有点太接地气，但中文似乎也没有短而好的表述方式。

**Review by kj415j45 — CHANGES_REQUESTED** [review]

File: wiki/Performance_points/zh.md (hunk 5 above)
Comment (YumeMuzi) [reply wiki/Performance_points/zh.md:69]: 先按你说的改吧

**Review by TPGPL — APPROVED** [review]

**Pull request merged**